src/data/explore-dapps.ts

```diff
@@ -0,0 +1,47 @@
+export interface Dapp {
+  name: string;
+  link: string;
+  tags: Array<string>;
+  description: string;
+  icon: string;
+  association: string;
+}
+
+const dApps: Array<Dapp> = [
+  {
+    name: 'Dracula',
+    link: 'https://app.dracula.fairdatasociety.org',
+    tags: ['text', 'editor', 'markdown', 'document'],
+    description:
+      'Markdown editor dapp that can help you create perfect readme files or documentation.',
+    icon: '',
+    association: 'md',
+  },
+  {
+    name: 'FairPhoto',
+    link: 'https://app.photo.fairdatasociety.org',
+    tags: ['photo', 'album'],
+    description: 'Photo album dapp.',
+    icon: '',
```

Comment:
@jayolin @tfius icons here right?

Reply:
Right now, the dApps content sit in this file, and the icons corresponding to each sit in the `public/media/dapps` folder. For example, for the NFTGenerator dApp, there's a corresponding `/public/media/dapps/NFTGenerator.svg` file that is displayed. The `icon` key was meant to do that, but since that's covered I'm going to have to remove it.

Reply:
Please do that, then I can review it again, thanks